demo/full/scripts/components/GenerateLinkURL.tsx

```diff
@@ -1,10 +1,23 @@
 import * as React from "react";
+import Button from "./Button";
 
 /**
  * @param {Object} props
  * @returns {Object}
  */
 function GeneratedLinkURL({ url }: { url: string | undefined | null }): JSX.Element {
+  const [hasBeenCopied, setHasBeenCopied] = React.useState(false);
+  const onCopyURLtoClipboard = () => {
```

Comment:
I don't know if you know this and it is here by no mean necessary, but as here the callback will most likely never change, we could rely on `React.useCallback` (https://react.dev/reference/react/useCallback).

Though I don't think this will change anything here beside making that code more footgun-y.

Though it's our demo page, we can be a little more daring here than we would be on ""real"" RxPlayer code :D